crates/sui/src/keytool.rs

```diff
@@ -686,6 +686,8 @@ impl KeyToolCommand {
             }
 
             KeyToolCommand::Unpack { keypair } => {
+                let keypair: SuiKeyPair = keypair.parse().unwrap();
```

Comment:
Maybe we can use `.expect("Expected a Base64 private key, but could not decode the input string to a Sui Keypair")` instead, or `?`.